settings.gradle.kts

```diff
@@ -195,3 +195,25 @@ gradle.settingsEvaluated {
         throw GradleException("This build requires JDK 11. It's currently ${getBuildJavaHome()}. You can ignore this check by passing '-Dorg.gradle.ignoreBuildJavaVersionCheck=true'.")
     }
 }
+
+// region platform include DSL
+
+fun includeSubproject(projectName: String) =
```

Comment:
For symmetry and maybe to make it easier to split things in the future, could we have a similar API for the unassigned/legacy layout?

e.g., instead of:
```
includeSubproject("foo")
```
We would have:
```
unassigned {
   subproject("foo")
}
```
The point being that when you're ready to move `foo` to `platforms/jvm`, you can just copy and paste the line.

Reply:
Done